packages/MarkdownEditor-Core.package/MarkdownInlineTextStyler.class/instance/postprocessTokens.st

```diff
@@ -0,0 +1,7 @@
+parsing
+postprocessTokens
```

Comment:
I'm not happy with the name. `postprocess` sounds like it is something that happens after everything is done. Doesn't make sense to me that we `interpret` after postprocessing. Maybe just call it `processAdjacentTokens`?